docs/source/commandline/commandline-all.rst

```diff
@@ -1626,85 +2260,119 @@ Core Misc Options
 .. _mame-commandline-drc:
 
 **-[no]drc**
-	Enable DRC (dynamic recompiler) CPU core if available for maximum speed. The default is ON (**-drc**).
+
+    Enable DRC (dynamic recompiler) CPU core if available for maximum speed.
+
+    The default is ON (**-drc**).
 
 .. _mame-commandline-drcusec:
 
 **\-drc_use_c**
 
-	Force DRC to use the C code backend. The default is OFF (**-nodrc_use_c**).
+    Force DRC to use the C code backend.
+
+    The default is OFF (**-nodrc_use_c**).
 
 .. _mame-commandline-drcloguml:
 
 **\-drc_log_uml**
 
-	Write DRC UML disassembly log. The default is OFF (**-nodrc_log_uml**).
+    Write DRC UML disassembly log.
+
+    The default is OFF (**-nodrc_log_uml**).
 
 .. _mame-commandline-drclognative:
 
 **\-drc_log_native**
 
-	write DRC native disassembly log. The default is OFF (**-nodrc_log_native**).
+    Write DRC native disassembly log.
+
+    The default is OFF (**-nodrc_log_native**).
 
 .. _mame-commandline-bios:
 
 **-bios** *<biosname>*
 
-	Specifies the specific BIOS to use with the current system, for systems that make use of a BIOS. The **-listxml** output will list all of the possible BIOS names for a system. The default is ``default``.
+    Specifies the specific BIOS to use with the current system, for systems that
+    make use of a BIOS. The **-listxml** output will list all of the possible
+    BIOS names for a system.
+
+    The default is ``default``.
 
 .. _mame-commandline-cheat:
 
 **-[no]cheat** / **-[no]c**
 
-	Activates the cheat menu with autofire options and other tricks from the cheat database, if present. The default is OFF (**-nocheat**).
+    Activates the cheat menu with autofire options and other tricks from the
+    cheat database, if present.
```

Comment:
Probably worth pointing out that this enables additional sliders for overclocking/underclocking various devices.  Also, save states may be incompatible with this option enabled vs disabled.

Reply:
Done.